jag-ccd-application/src/main/java/ca/bc/gov/open/ccd/controllers/UserController.java

```diff
@@ -81,7 +81,7 @@ public MapGuidToParticipantResponse mapGuidToParticipant(
             HttpEntity<MapGuidToParticipantResponse> resp =
                     restTemplate.exchange(
                             builder.toUriString(),
-                            HttpMethod.POST,
+                            HttpMethod.GET,
```

Comment:
Why this change?

Reply:
changed it back to HttpMethod.POST and also updated the CCD ORDS.